### PR TITLE
Bump to Version 1.1

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+#
+# Cut a libpod release.  Usage:
```
Comment (TomSweeneyRedHat) [inline release.sh]: buildah not libpod

File: release.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	sed -i "s/^\(Version: *\).*/\1${LOCAL_VERSION}/" contrib/rpm/buildah.spec
+}
+
+write_makefile_epoch()
+{
+	LOCAL_EPOCH="$1"
+	sed -i "s/^\(EPOCH_TEST_COMMIT ?= \).*/\1${LOCAL_EPOCH}/" Makefile
+}
+
+write_changelog()
```
Comment (TomSweeneyRedHat) [inline release.sh]: neat!  Much easier than the gronking I've done.

File: buildah.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ const (
 	Package = "buildah"
 	// Version for the Package.  Bump version in contrib/rpm/buildah.spec
 	// too.
-	Version = "1.0"
+	Version = "1.2-dev"
```
Comment (TomSweeneyRedHat) [inline buildah.go]: should this be 1.1 or does this get bumped up by one due to it being seen in master?

File: contrib/rpm/buildah.spec [hunk 4]
```diff
@@ -26,7 +26,7 @@
 
 Name:           buildah
 # Bump version in buildah.go too
-Version:        1.0
+Version:        1.2
```
Comment (TomSweeneyRedHat) [inline contrib/rpm/buildah.spec]: 1.1?

File: buildah.go (hunk 3 above)
Comment (rhatdan) [reply buildah.go]: There are two Patches here.  First one bumps to 1.1, second one to 1.2-dev.  We will tag and release the first one, the second one will just tell us if someone is running buildah off of master

When we release 1.2 we will do the same thing again.  This is how podman releases work.

Comment (TomSweeneyRedHat) [reply buildah.go]: Thanks for the 411.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Well as soon as I sent the LGTM, the Travis CI flipped from green to red on my screen....

**Comment by TomSweeneyRedHat** [comment]
Looks like the version.bats tests aren't happy with something.

```
not ok 160 buildah version current in .spec file Version
# (in test file ./version.bats, line 15)
#   `[ "$status" -eq 0 ]' failed
not ok 161 buildah version current in .spec file changelog
# (in test file ./version.bats, line 21)
#   `[ "$status" -eq 0 ]' failed
/home/travis/.travis/job_stages: line 169: 24709 Terminated              travis_jigger $! $timeout $cmd
```

**Comment by pixdrift** [comment]
@TomSweeneyRedHat, it's because the output from `buildah version` doesn't match what's in the changelog / spec file, which is correct behaviour for the test.

The version going in this commit should be stamped 1.1, then another commit to bump to 1.2?

**Comment by pixdrift** [comment]
The most recent changelog entry in the spec needs to be 1.2 to pass.

**Comment by rhatdan** [comment]
@pixdrift I am trying to allow adding -dev to version number so we can identify when people are testing from Master branch versus something we shipped.

**Comment by pixdrift** [comment]
@rhatdan, that's fine. But the check is checking that the latest changelog entry matches the version number returned by buildah. In this commit the version is 1.2-dev, but the latest commit in the spec file is 1.1-1

File: contrib/rpm/buildah.spec [hunk 5]
```diff
@@ -91,6 +91,99 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Sat Jun 9 2018 Dan Walsh <dwalsh@redhat.com> 1.1-1
```
Comment (pixdrift) [inline contrib/rpm/buildah.spec]: This line here needs 1.2-dev to pass the check as that is what buildah is outputting, really should be 1.2-dev-1... but anyway.

File: tests/version.bats [hunk 6]
```diff
@@ -11,12 +11,12 @@ load helpers
...
 	rversion=$(cat ${TESTSDIR}/../contrib/rpm/buildah.spec | awk '/^Version:/ { print $NF }')
-	run test "${bversion}" = "${rversion}"
+	run test "${bversion}" = "${rversion}" -o "${bversion}" = "${rversion}-dev"
 	[ "$status" -eq 0 ]
 }
 
 @test "buildah version current in .spec file changelog" {
 	bversion=$(buildah version | awk '/^Version:/ { print $NF }')
-	run bash -c "grep -A1 ^%changelog ${TESTSDIR}/../contrib/rpm/buildah.spec | grep -q \" ${bversion}-.*$\""
+	run bash -c "grep -A1 ^%changelog ${TESTSDIR}/../contrib/rpm/buildah.spec | grep -q \" ${bversion}.*$\""
```
Comment (pixdrift) [inline tests/version.bats]: Can I nit, and ask that this be reverted? ${bversion}- is correct, in this case 1.2-dev with -1. This was to stop people just pasting in the version number, when you should also include the package release number. I can revert it after the commit as I know this commit is something you want to get through.

**Comment by pixdrift** [comment]
It's quite peculiar that CentOS 7 and Fedora 27 passed on that RPM version check, but Fedora 28 failed.

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
Finally.
@rh-atomic-bot r=TomSweeneyRedHat

**Comment by pixdrift** [comment]
Well done :tada:

**Comment by pixdrift** [comment]
Looks good @rhatdan 👍 quite the changelog 😄

File: tests/rpm.bats [hunk 7]
```diff
@@ -167,7 +167,7 @@ load helpers
         echo "buildah version: $bv"
         echo "buildah rpm version: $rv"
         test $shortcommit = $id
-        test $bv = $rv
+	test $bv = ${rv} -o $bv = ${rv}-dev
```
Comment (TomSweeneyRedHat) [inline tests/rpm.bats]: :+1:

**Pull request merged**